### DISCUSSION TITLE
Does the column order matter for purposes of file validation?

### DISCUSSION THREAD

**Discussion by foustsWCH** [CMSgov/hospital-price-transparency#28] [Q&A] [answered]
Since the MRF file validator isn't available yet, I wanted to confirm a formatting question.  Does the _column order_ in the file matter for compliance purposes?  I'm asking because the order of the column descriptions in the README file specifications for the CSV Wide File is not the same as the order in the CSV Wide example file provided.  I assume this means that the column layout order does not matter so long as the column headers are named appropriately, but just want to confirm.

As an example - the README lists "additional_payer_notes | [payer_name] | [plan_name]"  at the very end of the required field list.  However, in the example excel file provided, this field typically appears grouped next to the other payer specific fields (negotiated dollar/negotiated percentage/negotiated algorithm/methodology) rather than at the far end of the file.

The README does the same thing with the field ""standard_charge | [payer_name] | [plan_name] | methodology" which is listed after the min/max columns on the README documentation, but in the example file is instead grouped next to the other payer specific fields (negotiated dollar/negotiated percentage/negotiated algorithm/methodology).

Thank you!

**Comment by RidenourK** [comment, marked as answer]
@foustsWCH Thank you for your question. 

Please refer to the [“Examples README”](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) on the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency). As indicated in the “Additional Notes” for the examples, the CSV examples present the data headers in row three in an order that differs from the CSV templates. Changing the order of the data headers will not generate a deficiency.
